<launch>
<!-- Run Map server -->
  
<arg name="map_file" default="$(find pmb2_grav_2dnav)/maps/map.yaml"/>
<node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

<node pkg="rviz" type="rviz" respawn="false" output="screen" name="rviz"
   	 args="-d '$(find pmb2_grav_2dnav)/rviz/mekaYPMB2.rviz'" required="true" />
<!-- Se etiquetan las maquinas en las que se lanza-->
<include file="$(find platform_bringup)/robots.machines" />
<!-- Incluimos la navegación en PMB2 suponiendo que ya está lanzado los sensores
-->
    <include file="$(find pmb2_grav_2dnav)/launch/navegacion_en_pmb2.launch" />

</launch>
